### PR TITLE
bpf: nodeport: trivial cleanups

### PR DESCRIPTION
**Pull request by julianwiedmann**
Two tiny cleanups I've been carrying for a while in a feature branch. Maybe they are helpful to others as well.

### CODE REVIEW

File: bpf/lib/nodeport.h [hunk 1]
```diff
@@ -761,8 +761,7 @@ int tail_nodeport_nat_egress_ipv6(struct __ctx_buff *ctx)
 					  info->sec_label,
 					  NOT_VTEP_DST,
 					  (enum trace_reason)CT_NEW,
-					  TRACE_PAYLOAD_LEN,
-					  (int *)&fib_params.l.ifindex);
```
Comment (aspsk) [inline bpf/lib/nodeport.h:765]: Isn't the `fib_params.l.ifindex` value used in the [consequent call](https://github.com/cilium/cilium/pull/24732/files#diff-304969c111195cf4253b336fc67c9891f17b91d49738a0cc334333ac11e84b9eL811) to `fib_redirect`?

Comment (aspsk) [reply bpf/lib/nodeport.h:765]: Ok, I see, there's `use_tunnel` flag below

**Review by aspsk — APPROVED** [review]

File: bpf/lib/nodeport.h (hunk 1 above)
Comment (julianwiedmann) [reply bpf/lib/nodeport.h:765]: exactly - maybe not so trivial after all, sorry :smile:. But even better to clean it up then, and clarify how that struct is used ...

**Comment by julianwiedmann** [comment]
The `ci-eks` fail is https://github.com/cilium/cilium/issues/24774.

**Pull request merged**